### ISSUE TITLE
Misleading {:error, :obscured}

### ISSUE THREAD

**Issue by opsb** [keathley/wallaby#280]
I got

```
{:error, :obscured} = Wallaby.start_session(metadata: metadata)
```

when trying to run a test with the selenium driver. In the selenium logs I got the error

```
java.util.concurrent.ExecutionException: org.openqa.selenium.WebDriverException: The best matching driver provider org.openqa.selenium.safari.SafariDriver can't create a new driver instance for Capabilities [{javascriptEnabled=true}]
```

indicating there's a problem with my installed drivers rather than an element being obscured. WebDriver is returning error `13` which is currently mapped to obscured at https://github.com/keathley/wallaby/blob/df1210215c76dd26499559c2db2f884894b3d8f3/lib/wallaby/httpclient.ex#L64.

**Comment by mhanberg** [comment]
This might be able to be closed due to #389.

**Comment by keathley** [comment]
Yup. I think we can close this now. At least we'll be returning real errors now.
